src/main/webapp/app/pages/CreateCompanyUsersPage.tsx

```diff
@@ -0,0 +1,418 @@
+        firstName: processedList[i + 1] || '',
+        lastName: processedList[i + 2] || '',
+        creationStatus: { status },
+      });
+    }
+  }
+
+  getErrorMessages() {
+    const failedUsers = this.userInfos.filter(
+      userInfo => userInfo.creationStatus.status === CreationStatus.ERROR
+    );
+    if (failedUsers.length !== 0) {
+      return (
+        <Alert variant="danger">
+          <div>
+            The following user(s) could not be created. Please review the
+            information and try again.
+          </div>
+          {failedUsers.map(userInfo => (
+            <div>
+              {userInfo.email}:{' '}
+              {getErrorMessage(
+                userInfo.creationStatus.error || new Error('error')
+              )}
+            </div>
+          ))}
+        </Alert>
+      );
+    }
+  }
+
+  getCreateUserModeTabs(): JSX.Element[] {
+    return Object.keys(CreateUserMode).map((mode: CreateUserMode) => {
+      return (
+        <Tab
+          className="mt-2"
+          eventKey={CreateUserMode[mode]}
+          title={CreateUserMode[mode]}
+        >
+          <h5>In Progress</h5>
+          {this.getErrorMessages()}
+          {this.createUserMode === CreateUserMode.BATCH && (
+            <AvField
+              name={'userList'}
+              label={'Users'}
+              type={'textarea'}
+              rows={5}
+              value={this.usersInputText}
+              placeholder={batchUserInputFormatText}
+              onChange={(event: any) => {
+                this.usersInputText = event.target.value;
+                this.parseUserInfoText();
+              }}
+              onKeyDown={this.onTabKey}
+            />
+          )}
+
+          {this.userInfos.map((info, idx) => (
+            <Row key={`${idx}`}>
+              <Col>
+                <AvField
+                  name={`email${idx}`}
+                  label={`Email ${idx + 1}`}
+                  value={this.userInfos[idx].email}
+                  onChange={(event: any) => {
+                    this.userInfos[idx].email = event.target.value;
+                  }}
+                  validate={{
+                    ...fieldRequiredValidation('email'),
+                    email: true,
+                    asyncLoginValidation: _.debounce(this.loginValidation, 500),
+                    duplicateEmailValidation: this.duplicateEmailValidation,
+                  }}
+                  disabled={CreateUserMode[mode] === CreateUserMode.BATCH}
+                />
+              </Col>
+              <Col>
+                <AvField
+                  name={`firstName${idx}`}
+                  label={'First Name'}
+                  value={this.userInfos[idx].firstName}
+                  onChange={(event: any) => {
+                    this.userInfos[idx].firstName = event.target.value;
+                  }}
+                  validate={{
+                    ...fieldRequiredValidation('first name'),
```

Comment:
Let's use SHORT_TEXT_VAL which was included in https://github.com/oncokb/oncokb-public/pull/788/files#diff-0cc27ecc1a8cb220a51e7c3eac69ba556cb73dc444fa9502816ec6c1cd1254b9R78